lib/takeaway.rb

```diff
@@ -0,0 +1,26 @@
+require 'message'
+
+class Takeaway
+  attr_reader :menu, :customer_name, :customer_number
+
+  def initialize menu
+    @menu = menu
+  end
+
+  def customer name, number
+    @customer_name = name
+    @customer_number = number
+  end
+
```

Comment:
Same thing ... If we want the takeaway to have a customer, why don't use the customer object instead of strings. The customer class becomes redundant... We should be dealing only with objects and passing them around ...